### ISSUE TITLE
Incorrect output of shellcraft.amd64.popad

### ISSUE THREAD

**Issue by david942j** [Gallopsled/pwntools#848]
pwnlib/shellcraft/templates/amd64/popad.asm
```
    pop rdi
    pop rsi
    pop rbp
    pop rsp
    pop rbp <-- should be rbx !
    pop rdx
    pop rcx
    pop rax
```

**Comment by david942j** [comment]
In addition, I think `pop rsp` should be changed to `add rsp, 8` ?

**Comment by zachriggle** [comment]
Fixed via #853
